src/AppBundle/Entity/ProcurationProxy.php

```diff
@@ -612,7 +613,7 @@ public function getVoteOffice(): ?string
         return $this->voteOffice;
     }
 
-    public function setVoteOffice(?string $voteOffice)
+    public function setVoteOffice(string $voteOffice): void
```

Comment:
Are you sure this will work? The Form component may inject `null` if not configured and no string is provided. Can you check?